### DISCUSSION TITLE
SDLC Member Feedback: PR #1951 Fixes StartService RPC not ACKed on IVI boot

### DISCUSSION THREAD

**Discussion by theresalech** [smartdevicelink/sdl_ios#1981] [General]
The SDLC review period for PR #1951 starts now and continues through May 17, 2021. Following the review period, the Project Maintainer will conduct final testing to prepare the PR for release in Hotfix 7.1.1.

Please leave any any questions, comments, or other feedback about PR #1951 and/or Issue #1783 here, so that necessary discussion can take place as soon as possible.

Thank you!

**Comment by theresalech** [comment]
The Project Maintainer believes Issue #1795 can also be closed by PR #1951. Please consider this in your testing and review as well, and provide any feedback here. Thanks!

**Reply by stefanek22** [reply]
Agreed from the Ford side

**Comment by mrapitis** [comment]
We have integrated PR #1951 into 15 sample apps and connectivity has been positive.  
We have tested multiple scenarios including iAP over BT and USB as well as transport switching between the two. 
Our testing shows that issues #1783 and #1795 have been resolved with the changes included in this branch.
Thus far no regressions or degradation to stability have been observed.

**Comment by joeljfischer** [comment]
The SDLC review period has concluded for this PR. It will be merged and integrated into the next HotFix release. Thank you for testing!
